packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -587,6 +701,27 @@ const initialize = async () => {
 	}
 };
 
+const useSuggestedConfig = (config: ModelConfiguration) => {
```

Comment:
If we are using suggested config this should probably take its name and description too?
Seems weird from user POV when it doesnt IMO

Reply:
I agree, this should is an excellent question, but UX related.
Let's move that to another Issue for improvement.